internal/csi-addons/rbd/replication.go

```diff
@@ -814,9 +814,9 @@ func getLastSyncInfo(description string) (*replication.GetVolumeReplicationInfoR
 		return nil, fmt.Errorf("no snapshot details: %w", corerbd.ErrLastSyncTimeNotFound)
 	}
 	type localStatus struct {
-		LocalSnapshotTime    int64 `json:"local_snapshot_timestamp"`
-		LastSnapshotBytes    int64 `json:"last_snapshot_bytes"`
-		LastSnapshotDuration int64 `json:"last_snapshot_sync_seconds"`
+		LocalSnapshotTime    int64  `json:"local_snapshot_timestamp"`
+		LastSnapshotBytes    int64  `json:"last_snapshot_bytes"`
+		LastSnapshotDuration *int64 `json:"last_snapshot_sync_seconds"`
```

Comment:
if `last_snapshot_sync_seconds` can be empty when Ceph returns this, can `last_snapshot_bytes` not also be empty?

Reply:
> if `last_snapshot_sync_seconds` can be empty when Ceph returns this, can `last_snapshot_bytes` not also be empty?

Maybe but currently it does not matter since there's no *int64 type in protoc3 sadly. 

Anyway csiaddons defines bytes 0 as empty and sets it to nil in status. 
This pr just rectifies the nil case for last sync duration. 
 
> Is it known what Ceph versions return incomplete results?
I asked the same too in previous pr but didn't get an answer.